src/reader_test.go

```diff
@@ -23,7 +23,7 @@ func TestReadFromCommand(t *testing.T) {
 
 	// Normal command
 	reader.fin(reader.readFromCommand(nil, `echo abc && echo def`))
-	if len(strs) != 2 || strs[0] != "abc" || strs[1] != "def" {
+	if len(strs) != 2 || strs[0] != util.OS.Sieve("abc", "abc ") || strs[1] != "def" {
```

Comment:
I would simply change the command so that it yields the same result on both systems.

i.e. `echo abc&&echo def`